Godeps/_workspace/src/github.com/steakknife/hamming/README.md

```diff
@@ -0,0 +1,3 @@
+Copyright (c) 2014 Barry Allard
+
+MIT license
```

Comment:
@jbenet Tyrannical, bureaucratic "standards" people exist to foist unnecessary bikeshedding FUD and work on others while contributing no real value.  I refuse to set this precedent unless you have a really good reason.